test-suite/testkit/src/api.rs

```diff
@@ -57,6 +59,7 @@ impl fmt::Display for ApiKind {
         match self {
             ApiKind::System => write!(f, "api/system"),
             ApiKind::Explorer => write!(f, "api/explorer"),
+            ApiKind::Runtime => write!(f, "api/runtimes/rust"),
```

Comment:
Well, since it corresponds to `runtimes/rust`, I think `ApiKind::RustRuntime` will be better.